app/v2/models/response/retrieveHistoricNonFhlUkPiePeriodSummary/PeriodExpenses.scala

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v2.models.response.retrieveHistoricNonFhlUkPiePeriodSummary
+
+import play.api.libs.functional.syntax.toFunctionalBuilderOps
+import play.api.libs.json.{ JsPath, Json, OWrites, Reads }
+
+case class PeriodExpenses(premisesRunningCosts: Option[BigDecimal],
+                          repairsAndMaintenance: Option[BigDecimal],
+                          financialCosts: Option[BigDecimal],
+                          professionalFees: Option[BigDecimal],
+                          costOfServices: Option[BigDecimal],
+                          other: Option[BigDecimal],
+                          travelCosts: Option[BigDecimal],
+                          residentialFinancialCost: Option[BigDecimal],
+                          residentialFinancialCostsCarriedForward: Option[BigDecimal],
+                          consolidatedExpenses: Option[BigDecimal],
```

Comment:
Can we move consolidatedExpenses filed above travelCosts to keep it consistent with the tech spec. Its trivial and does not affect functionality but helps with reviewing with tech specs.